examples/http_headers/README.md

```diff
@@ -1,6 +1,15 @@
 ## http_headers
 
-this example handles http request/response headers events and log all headers.
+This example handles http request/response headers events and log all headers.
+
+In case there is information in the argument it will add the header to the response: 
```

Comment:
I don't believe the custom header is optional -- the plugin configuration fails if not present.

```suggestion
In envoy.yaml, the custom header is given as the plugin configuration like the following:

```yaml
                          configuration:
                            "@type": type.googleapis.com/google.protobuf.StringValue
                            value: |
                              {
                                "header": "x-wasm-header",
                                "value": "demo-wasm"
                              }
```

and this adds the `x-wasm-header: demo-wasm` header to all the responses.
```